pristine-zen-garden.html

```diff
@@ -0,0 +1,201 @@
+        <a href="http://mezzoblue.com/zengarden/faq/#aaa" title="Read about the accessibility of this site" class="zen-accessibility">A11y</a>
+        <a href="https://github.com/mezzoblue/csszengarden.com" title="Fork this site on Github" class="zen-github">GH</a>
+      </footer>
+
+    </div>
+
+    <aside class="sidebar" role="complementary">
+
+      <div class="wrapper">
+
+        <div class="design-selection" id="design-selection">
+          <h3 class="select">Select a Design:</h3>
+
+          <nav role="navigation">
+            <ul>
+              <li> 
+                <a href="/205/" class="design-name">spring360</a> by <a href="http://www.medialab360.com" class="designer-name">Rene Hornig</a> 
+              </li>  
+              <li> 
+                <a href="/204/" class="design-name">Withering Beauty</a> by<a href="http://www.wduffy.co.uk/" class="designer-name">William Duffy</a>
+              </li>         
+              <li> 
+                <a href="/203/" class="design-name">Tiny Blue</a> by <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
+              </li>       
+              <li> 
+                <a href="/202/" class="design-name">Retro Theater</a> by            <a href="#" class="designer-name">Eric Rogé</a> 
+              </li>         
+              <li>
+                <a href="/201/" class="design-name">Lily Pond</a> by <a href="#" class="designer-name">Rose Thorogood</a>
+              </li>         
+              <li>
+                <a href="/200/" class="design-name">Icicle Outback</a> by           <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
+              </li>        
+              <li>
+                <a href="/199/" class="design-name">CSS ZEN ARMY</a> by           <a href="http://www.niceguy.com" class="designer-name">Carl Desmond</a>
+              </li>         
+              <li>
+                <a href="/198/" class="design-name">The Original</a> by <a href="#" class="designer-name">Joachim Shotter</a>
+              </li>
+            </ul>
+          </nav>
+
+        </div>
+
+        <div class="design-archives" id="design-archives">
+          <h3 class="archives">Archives:</h3>
+          
+          <nav role="navigation">
+            <ul>
+              <li class="next">
+                <a href="/214/page3">
+                Next Designs <span class="indicator">&rsaquo;</span> </a>
+              </li>
+              <li class="previous"> <a href="/214/page1/">
+                <span class="indicator">&lsaquo;</span> Previous Designs              </a>
+              </li>
+              <li class="viewall">
+                <a href="http://www.mezzoblue.com/zengarden/alldesigns/" title="View every submission to the Zen Garden."> View All Designs </a>
```

Comment:
Nice indentation! I don't see any major problems with it.. Just lines 157-158 could probably be on one line.

Also, while this probably isn't part of the assignment, you might want to fix the weird text spacing on 132, 138, 141, and 161